test/lazy-loading.test.js

```diff
@@ -123,6 +123,22 @@ describe('lazy loading', () => {
         expect(comboBox.selectedItem).to.be.null;
       });
 
+      describe('when autoOpenDisabled', () => {
+        beforeEach(() => (comboBox.autoOpenDisabled = true));
+
+        (isComboBoxLight ? it.skip : it)('should not clear filter when loading first page', () => {
```

Comment:
The test is passing also for `vaadin-combo-box-light` so let's not skip it.

```suggestion
        it('should not clear filter when loading first page', () => {
```

Reply:
Actually, it fails.